src/utils/history.ts

```diff
@@ -0,0 +1,116 @@
+import { AxiosResponseWithExtraData } from '../interfaces/request'
+import { Probe } from '../interfaces/probe'
+import { log } from 'console'
+import path from 'path'
+
+const sqlite3 = require('sqlite3').verbose()
+
+export type HistoryLogType = {
+  id: number
+  probeID: string
+  statusCode: number
+  probeURL: string
+  responseTime: number
+}
+
+let db: any
+
+/**
+ * createTable will create the history table if it does not exist
+ */
+async function createTable() {
+  const createTablSQL = `CREATE TABLE IF NOT EXISTS history (
```

Comment:
typo: `createTablSQL`, supposed to be `createTabelSQL`

Reply:
fixed. thank you!